hooks/useWeatherData.ts

```diff
@@ -255,6 +252,68 @@ export const useWeatherData = (city: string, today: boolean) => {
     return { feelsLike, humidity, visibility, windSpeed, pressure };
   };
 
+  const filterWeatherDataForNextDays = () => {
+    if (!forecastData) return [];
+
+    const weatherDataForNextDays = [];
+
+    for (let i = 1; i <= 4; i++) {
+      const dayStart = new Date(
+        tomorrowMidnight.getTime() + (i - 1) * oneDayInMilliseconds,
+      );
+      const dayEnd = new Date(dayStart.getTime() + oneDayInMilliseconds);
+
+      const dayData = forecastData.list.filter((item: ForecastItem) => {
+        const itemDate = new Date(item.dt * 1000);
+        return itemDate >= dayStart && itemDate < dayEnd;
+      });
+
+      weatherDataForNextDays.push(dayData);
+    }
+
+    return weatherDataForNextDays;
+  };
+
+  const weatherDataForNextFourDays = filterWeatherDataForNextDays();
+
+  const getDayWeatherSummary = (number: number): DayWeatherSummary => {
```

Comment:
Nie podoba mi się, że `getDayWeatherSummary(0)` oznacza "daj mi pogodę na jutro" - nieintuicyjne to. Trzeba pomyśleć jak zmienic ten parametr/nazwę funkcji tak, zeby to miało ręce i nogi, albo w najgorszym wypadku chociaż zostawić jakiś komentarz